lib/containers/SendReceive.js

```diff
@@ -140,6 +165,7 @@ class SendReceive extends PureComponent {
   }
 
   // only render tabs if a wallet and account has been selected
+  // TODO: only render dropdown if standard wallet
```

Comment:
the account selection dropdown should only render if its a standard wallet since there is only 1 account for multisig wallets